test/infrastructure/docker/api/v1beta1/dockercluster_types.go

```diff
@@ -65,6 +65,9 @@ type DockerLoadBalancer struct {
 	// node is added or removed. The template will also support the JoinHostPort function to join the host and port of the backend server.
 	// +optional
 	CustomHAProxyConfigTemplateRef *corev1.LocalObjectReference `json:"customHAProxyConfigTemplateRef,omitempty"`
+
+	// Disable allows skipping the creation of the cluster load balancer.
+	Disable bool `json:"disable,omitempty"`
```

Comment:
Not completely happy with the wording ("disable"), should it be `disabled`? because it does not describe an action.

Also please use the godoc to document when this should get used and maybe also a disclamer similar to how we did for the CustomHAProxyConfigTemplateRef.